search-app/test/cmr/search/test/data/metadata_retrieval/revision_format_map.clj

```diff
@@ -89,16 +90,32 @@
             #{:echo10 {:format :umm-json
                        :version "1.3"}})))))
 
+(defn- drop-in
```

Comment:
I think we already have function `cmr.common.util/dissoc-in` that does the same thing and better. Let's use the existing function.

Reply:
very cool, I would love to use an existing function